pypeman/contrib/http.py

```diff
@@ -246,3 +246,17 @@ class RequestNode(HttpRequest):
     def __init__(self, *args, **kwargs):
         warnings.warn("RequestNode node is deprecated. New name is 'HttpRequest' node", DeprecationWarning)
         super().__init__(*args, **kwargs)
+
+
+class HttpJsonRequest(nodes.HttpRequest):
```

Comment:
Changed ! 
notes: 
- I removed encoding because it's not useful. autodetected and in 3.9 json lib will drop param. 
- since 3.6 json can loads bytes